sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -111,6 +112,35 @@ class Analyzer(
 
   override protected def defaultCatalogName: Option[String] = conf.defaultV2Catalog
 
+  private lazy val v2SessionCatalog = new V2SessionCatalog(catalog, conf)
```

Comment:
minor and not a propose to change, just a quick check that the naming `V2SessionCatalog` or `SessionCatalogV2` ? 

I find V2 at the end like `WriteToDataSourceV2` or `CatalogV2Implicits` or `DataSourceV2Relation` in current code space, but also sometimes `V2SessionCatalog`, so I guess either way is ok?